src/libstd/thread/local.rs

```diff
@@ -105,10 +105,12 @@ pub struct LocalKey<T> {
 #[allow_internal_unstable]
 macro_rules! thread_local {
     (static $name:ident: $t:ty = $init:expr) => (
+        #[cfg_attr(not(stage0), allow(static_could_be_const))]
```

Comment:
these might have interior mutability... The error messages the user would get would be quite confusing, I'm not sure what to do. for now they are simply ignored